Src/FluentAssertions.Dotnet/project.json

```diff
@@ -1,17 +1,12 @@
 ﻿{
   "supports": {
-    "net46.app": {},
-    "uwp.10.0.app": {},
-    "dnxcore50.app": {}
   },
   "dependencies": {
-    "Microsoft.NETCore": "5.0.0",
-    "Microsoft.NETCore.Portable.Compatibility": "1.0.0",
```

Comment:
Superseded by using NETStandard.Library. The compatibility one is only needed for referencing older PCL's.